scripts/west_commands/build.py

```diff
@@ -24,6 +24,7 @@
 
 BUILD_USAGE = '''\
 west build [-h] [-b BOARD[@REV]]] [-d BUILD_DIR]
+           [-S SNIPPET]
```

Comment:
Thanks for noticing, but it's plural in the `west help build` output because the dest variable is in plural:
```
$ west help build
usage: west build [-h] [-b BOARD[@REV]]] [-d BUILD_DIR]
...
   -S SNIPPETS, --snippet SNIPPETS
                        add the argument to SNIPPET; may be given multiple times. Forces CMake to run again if given. Do not use this option with manually specified -DSNIPPET... cmake arguments: the results are undefined

```

Ref: https://github.com/zephyrproject-rtos/zephyr/blob/3412e55d183ea098f7420da485d76902bafbc05f/scripts/west_commands/build.py#L139


I have no preference regarding singular or plural, only regarding consistency :)

Wonder if we can get the detailed help text to be singular, so they match.

Reply:
Good catch, thanks. I have added `metavar` arguments to use singular form for both SNIPPET and SHIELD.